frontend/src/pages/eventDetail/EventDetail.js

```diff
@@ -30,39 +32,26 @@ export const EventDetail = () => {
 
       {eventState.data && (
         <>
-          <Heading size="xl" className="self-center pb3">
-            {t('Page.EventDetail.Heading', { name: eventState.data.name })}
-          </Heading>
-
-          <Layout flex flex-row>
-            <Layout pt3 pr3 flex flex-row justify-between w-50>
-              <Label>{t('Page.EventDetail.EventStartDateLabel')}</Label>
-
-              <span className={classNames('b')}>
-                {formatDate(eventState.data.startDate)}
-              </span>
-            </Layout>
-
-            <Layout pt3 pl3 flex flex-row justify-between w-50>
-              <Label>{t('Page.EventDetail.EventEndDateLabel')}</Label>
-
-              <span className={classNames('b')}>
-                {formatDate(eventState.data.endDate)}
-              </span>
-            </Layout>
-          </Layout>
-
-          <Layout pt3>
-            <span>{eventState.data.description}</span>
-          </Layout>
-
-          <Button
-            className="w-50 mv3 self-center"
-            primary
-            onClick={onOpenModalClick}
+          <Jumbo
+            header={t('Page.EventDetail.Heading', {
+              name: eventState.data.name,
+            })}
+            mainBody={eventState.data.description}
+            footer={
+              <Button color="primary" onClick={onOpenModalClick}>
+                {t('Page.EventDetail.AddNewLobbyButton')}
+              </Button>
+            }
           >
-            {t('Page.EventDetail.AddNewLobbyButton')}
-          </Button>
+            <Paragraph>
+              {t('Page.EventDetail.EventStartDateLabel')}
+              {formatDate(eventState.data.startDate)}
```

Comment:
Kdyz to neni ani jedno obaleno v zadnym dalsim HTML elementu, tak tohle je divny - dal bych to jako preklad s replacementem - datumem.. Aby se tam dal dat nejaky delimiter - :, - , atd..